src/tests/admin/conferenceRoom/ForARoomDisabledTheTabletDoesNotScheduleMeetings.java

```diff
@@ -0,0 +1,58 @@
+package tests.admin.conferenceRoom;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.util.List;
+import java.util.Map;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import framework.pages.admin.HomeAdminPage;
+import framework.pages.admin.conferencerooms.RoomsPage;
+import framework.pages.tablet.HomeTabletPage;
+import framework.pages.tablet.SearchPage;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC24: Verify that when a room status is disabled, the tablet does not schedule meetings
+ * @author Juan Carlos Guevara
+ */
+public class ForARoomDisabledTheTabletDoesNotScheduleMeetings {
+	ExcelReader excelReader = new ExcelReader(EXCEL_INPUT_DATA);
+	List<Map<String, String>> testData = excelReader.getMapValues("Resources");
+	String roomName = testData.get(1).get("Room Name");
+
+	@BeforeClass
+	public void precondition(){
+
+		//Disable room in Admin 
+		HomeAdminPage homeAdminPage = new HomeAdminPage();
```

Comment:
It should be a global variable, because you use the same variable on your afterClass, please correct that